docs/source/model_design/risk_factors.rst

```diff
@@ -339,6 +339,33 @@ complexity into the exposure component.
 What is a risk effect?
 ++++++++++++++++++++++
 
+A **risk effect** is a differential outcome of some risk exposure for which 
+the exposure is causally and measurably related to the outcome.
+
+For example, we might model the risk effect *blood pressure* as an outcome of 
+*sodium intake*. For each measurement of the exposure sodium intake, this model 
+would map to a probability distribution of possible blood pressures.
+
+.. note::
+  Wrting this up, I realized I'm not sure what the definition is. In the above, 
+  is the risk outcome "blood pressure"? is it a given measurement of blood 
+  pressure? or is it a differential probability of a given measure of blood 
+  pressure?
+
```

Comment:
For risk effect outcome, we usually measure the proportion or burden of disease attributable to specific risk factor. Talking about _blood pressure_, it reminds me the casual relationship of high sodium intake -> high blood pressure -> ischemic heart disease. To model a risk-outcome pair, we have to find evidence from epidemiological studies supporting a causal relationship between risk factor exposure and the outcome, as well as available data to estimate effect sizes per unit of exposure. Evidence that these effects can be generalized to a general population.